src/oemof/tabular/facades.py

```diff
@@ -402,30 +466,27 @@ class Dispatchable(Source, Facade):
     ...         'min': 0.2})
 
     """
+    bus: Bus
 
-    def __init__(self, *args, **kwargs):
-        kwargs.update({"_facade_requires_": ["bus", "carrier", "tech"]})
-        super().__init__(*args, **kwargs)
+    carrier: str
 
-        self.profile = kwargs.get("profile", 1)
+    tech: str
 
-        self.capacity = kwargs.get("capacity")
+    profile: float = 1
```

Comment:
How to describe this? float or sequence of floats

Reply:
`Union[float, list[float]]`?

Reply:
I think so!